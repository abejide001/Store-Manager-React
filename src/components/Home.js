import React from 'react';
import '../assets/css/style.css';
import Footer from './layouts/Footer';
import Header from './layouts/Header';
import HomeBody from './layouts/HomeBody';

export default function Home() {
  return (
    <div className="landing-page">
      <Header />
      <HomeBody />
      <Footer />
    </div>
  );
}
