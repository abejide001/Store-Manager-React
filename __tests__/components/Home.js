/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Home from '../../src/components/Home';

describe('<Home />', () => {
  describe('render()', () => {
    test('renders the component', () => {
      const wrapper = shallow(<Home />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
