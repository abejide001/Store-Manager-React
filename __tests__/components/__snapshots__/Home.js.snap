// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> render() renders the component 1`] = `
<div>
  <h1>
    This is home file
  </h1>
</div>
`;
